content/blog/package-react-components/index.md

```diff
@@ -0,0 +1,776 @@
+---
+title: Package, Publish and Distribute React Components in NPM
+date: "2020-01-18"
+author: franleplant
+description:
+  "In this post we explore in detail how to share React Components which are typically made of a
+  bundle of Javascript, CSS (styles), Images, etc.
+  We explore how to package and bundle the code with Webpack so that it can be consumed appropriately,
+  and we explore how to publish and distribute it to NPM so that it can be easily accessible either
+  via the public NPM repository or a private repository.
+  We also analyse different theming strategies for larger Component Libraries and UI Frameworks.
```

Comment:
Super nit/: _Analyze_ for US audiences according to this: https://writingexplained.org/analyse-or-analyze-difference

 _You should use analyse with primarily British audiences, and analyze with primarily American audiences. Using the nonstandard variant will be distracting to your readers, and detract from your writing._

Reply:
Will push in the following commits